sotopia/generation_utils/generate.py

```diff
@@ -334,7 +334,27 @@ def obtain_chain(
             openai_api_base="https://api.groq.com/openai/v1",
             openai_api_key=os.environ.get("GROQ_API_KEY"),
         )
-        chain = LLMChain(llm=chat_openai, prompt=chat_prompt_template)
+        chain = chat_prompt_template | chat_openai
+        return chain
+    elif "azure" in model_name:
+        # azure|deployment_name|version|endpoint|azure_openai_api_key
+        azure_credentials = model_name.split("|")[1:]
```

Comment:
Use "/" instead of "|" to make it consistent with other models?

Reply:
I tried to use "|" instead of "/" because the endpoint is a url type of string e.g. "http://azure.xxx"

Reply:
maybe you can use `resource_name`, which can be passed as `f"https://{resource_name}.openai.azure.com/`?